app/controllers/nonprofits/refunds_controller.rb

```diff
@@ -25,7 +24,10 @@ def index
 private
 
     def charge_params
-      params.require(:charge_id, refund: [:amount])
+      {
+        :charge_id => params.require(:charge_id),
+        :refund => params.require(:refund).permit(:amount)
+      }
```

Comment:
You'll likely also need this change.

```suggestion      
  params.require(refund: :amount)
```

Reply:
I've now updated the `create` method and inlined the unnecessary extra `charge_params` method. The `require`/`permit` methods are making more sense to me now - I gather that these are used to clean the incoming HTTP POST data to make sure it has the items you expect and only those.